css/index.css

```diff
@@ -137,10 +175,103 @@ footer nav {
     justify-content: space-between;
     align-items: center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: 1.4rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+@media (min-width: 1000px){
+	.middle-content .boxes .Box2 {
+	background: forestgreen;  
+}
+.middle-content .boxes .Box4{
+	background: darkorchid; }
+	.middle-content .boxes .Box6 {
+	background: hotpink; 
+	}
+.middle-content .boxes .Box8 {
+	background: indigo; 
+}
+.middle-content .boxes .Box10 {
+	background: lawngreen; 
+}
+}
+
+@media (max-width: 768px){
+	.container{
+
+		width: 100%; 
+        padding: 5px; 
+        margin: 0 auto;
+         
+    }
+    header {
+    	width: 100%; 
+    	display: flex; 
+    	flex-stretch: stretch; 
+    	justify-content: center; 
+    }
+
+    header nav {
+    	width: 100%; 
+    }
+    .top-content {
+    	width: 100%; 
+    }
+    .middle-content .boxes .box{
+        min-width: 100px; 
+    }
+	.middle-content .boxes .Box1 {
+		background: teal;  
+	}
+	.middle-content .boxes .Box3 {
+	background: gold; }
+	.middle-content .boxes .Box5{
+	background: cadetblue; 
+	}
+	.middle-content .boxes .Box7 {
+	background: coral; 
+	}
+	.middle-content .boxes .Box9 {
+	background: crimson; 
+	}
+	}
+
+@media (max-width: 400px){
+    .container {
+        width: 100%; 
+        padding: 2px; 
+        margin: auto; 
+    }
+    header {
+    	width: 100%; 
+        flex-wrap: wrap; 
+        flex-direction: column; 
+      } 
+      
```

Comment:
Your image is left aligned currently since you haven't aligned it yet! Overwrite the position of your image by doing something along the lines of 
header img {
 align: center;
}